backend/src/nodes/utils/torch_types.py

```diff
@@ -18,3 +19,23 @@
     GFPGANv1Clean,
     RestoreFormer,
 ]
+
+
+def isPyTorchSRModel(model: PyTorchSRModel):
+    return isinstance(model, PyTorchSRModels)
+
+
+PyTorchFaceModels = (GFPGANv1Clean, RestoreFormer)
+PyTorchFaceModel = Union[GFPGANv1Clean, RestoreFormer]
+
+
+def isPyTorchFaceModel(model: PyTorchFaceModel):
+    return isinstance(model, PyTorchFaceModels)
+
+
+PyTorchModels = (*PyTorchSRModels, *PyTorchFaceModels)
+PyTorchModel = Union[PyTorchSRModel, PyTorchFaceModel]
+
+
+def isPyTorchModel(model: PyTorchModel):
+    return isinstance(model, PyTorchModels)
```

Comment:
From its type, you already know that `model` is a pytorch model, so this function should always return true. Of course, that's not what the function is supposed to do so let's fix its type. 

In general, the point of these type check functions is that they take a value of some type and return whether the current value is actually a most specific type. The best example is `isinstance(x, t)`. This function tells you whether a value x (of any type) is also of type `t`. The key point here is that the input value `x` has a broader type than the type being checked. E.g. `isinstance("foo", str)` doesn't make sense to check in code because we already know that `"foo"` is a `str`.

So how broad can we go for `model`? As far the implementation of `isPyTorchModel` model allows us.
Since we only use `isinstance` to implement `isPyTorchModel`, we can allow anything that `isinstance` allows and `isinstance` allows everything. Literally any value.

Same for `isPyTorchFaceModel` and `isPyTorchSRModel` as well.

```suggestion
def isPyTorchModel(model: object):
    return isinstance(model, PyTorchModels)
```